versions/3.0.3.md

```diff
@@ -317,15 +317,15 @@ url: https://www.apache.org/licenses/LICENSE-2.0.html
 
 #### <a name="serverObject"></a>Server Object
 
-An object representing a Server.
+An object representing a single server or a group of servers sharing a common URL structure.
 
 ##### Fixed Fields
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="serverUrl"></a>url | `string` | **REQUIRED**. A URL to the target host.  This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served. Variable substitutions will be made when a variable is named in `{`brackets`}`.
+<a name="serverUrl"></a>url | `string` | **REQUIRED**. A URL to the target host.  This URL may include variable expressions, delimited by `{`curly braces`}`.  Each variable expression MUST correspond to a [Server Variable Object](#serverVariableObject) named in the `variables` field, and the API consumer MUST replace each variable expression with a with a valid value, as specified by the corresponding Server Variable Object.  The URL MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served.
 <a name="serverDescription"></a>description | `string` | An optional string describing the host designated by the URL. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
-<a name="serverVariables"></a>variables | Map[`string`, [Server Variable Object](#serverVariableObject)] | A map between a variable name and its value.  The value is used for substitution in the server's URL template.
+<a name="serverVariables"></a>variables | Map[`string`, [Server Variable Object](#serverVariableObject)] | A map between a server variable name and a Server Variable Object.  Each server variable name SHOULD be referenced by a variable expression in the `url`.
```

Comment:
... not disallowing unused server variables. But server variables don't serve any purpose unless they're used. In an editor, a warning on an unused server variable might alert the user an oversight, so I thought it'd be useful to note this here, with the SHOULD clause.

Reply:
I'd rather not include the last sentence.  I don't want to set the precedent that everywhere we have something that can be used we have to state where it can be used.